packages/adapter-surrealdb/tests/index.test.ts

```diff
@@ -1,18 +1,19 @@
-import Surreal from "surrealdb.js"
+import Surreal, { ExperimentalSurrealHTTP } from "surrealdb.js"
 import { runBasicTests } from "@auth/adapter-test"
+import fetch from "node-fetch"
```

Comment:
can we not use node-fetch here? tests were passing for me even without it.

Reply:
For this PR we need to wait for surrealdb.js to merge this other PR: https://github.com/surrealdb/surrealdb.js/pull/171

I tested it using my fork with the fix

Reply:
`node-fetch` (or any other `fetch` polyfill) indeed is not necessary anymore, as all Node.js versions have `fetch` already available.